CoreSchemaView/release/modules/ext/docs/CoreSchemaView/src/au/gov/asd/tac/constellation/views/schemaview/schema-view.md

```diff
@@ -40,33 +40,6 @@ View" />
 
 </div>
 
-## Attributes
```

Comment:
If we're removing all the sub-panels from the main page, it might be nice if we could still make a mention of them here (at least indicating that there are a bunch of tabs to be explored), maybe linking to the respective pages